StanwoodDialog.podspec

```diff
@@ -13,11 +13,16 @@ This allows a more personal approach and dismisses automatically when no interac
   s.author           = { 'stanwood' => 'ios.frameworks@stanwood.io' }
   s.source           = { :git => 'https://github.com/stanwood/StanwoodDialog_iOS.git', :tag => s.version.to_s }
 
-  s.ios.deployment_target = '10.0'
+  s.ios.deployment_target = '10.3'
```

Comment:
@aaronTredrea any reason we had to bump to 10.3? I would keep it as 10.0 if there are no tradeoffs

Reply:
I am pretty sure there was a reason for this, but it was a while ago so I will roll back and test now to check

Reply:
`requestReview()' is only available in iOS 10.3 or newer`

Reply:
Seemed like a fair sacrifice as as far as I can see - 10.X users take up 0.7% of ipad and iphone users.... our restriction likely only affects <0.3% of all iOS users

Reply:
and of the number of people that will use this pod.... likely 0%